packages/backend/src/app.ts

```diff
@@ -177,6 +179,7 @@ const PAYMENT_POINTER_PATH = '/:paymentPointerPath+'
 export interface AppServices {
   logger: Promise<Logger>
   knex: Promise<Knex>
+  axios: Promise<AxiosInstance>
```

Comment:
do we need this? I'm not seeing this referenced, nor axios added in the ioc container

Reply:
Ah, it's used in the follow up PR: https://github.com/interledger/rafiki/pull/1857/files#diff-2ef3cf9bf12d21de5f65f0c473654ec6d62ac76a214fbf53a4afa2d3a92e7bf3

This is the result of my splitting of changes into PRs...

Reply:
Updated